games-roguelike/tome/files/tome-2.3.5-format.patch

```diff
@@ -1,5 +1,5 @@
---- dungeon.c.old	2016-02-07 20:12:39.258150130 +0100
-+++ dungeon.c	2016-02-07 20:12:58.239805610 +0100
+--- a/dungeon.c	2016-02-07 20:12:39.258150130 +0100
```

Comment:
I normally [drop](https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html#clean-patch-howto) the dates